test/setup/main.tf

```diff
@@ -56,16 +63,23 @@ module "gke-project-2" {
     "cloudkms.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "container.googleapis.com",
+    "compute.googleapis.com",
     "pubsub.googleapis.com",
     "serviceusage.googleapis.com",
     "storage-api.googleapis.com",
   ]
+  activate_api_identities = [
```

Comment:
@marko7460 Did you get an error without this? We should only need this if are using this identity cross project (which we are not) or we need to apply additional IAM roles on top of the existing ones (in which case we will need to supply `roles/container.serviceAgent` + whatever additional roles we need.)

Reply:
I was getting the error ```Step #1 - "prepare": [1m[31mError: [0m[0m[1mRequest "Set IAM Binding for role \"roles/cloudkms.cryptoKeyEncrypterDecrypter\" on \"project \\\"ci-gke-39e386ee-bb70\\\"\"" returned error: Batch request and retried single request "Set IAM Binding for role \"roles/cloudkms.cryptoKeyEncrypterDecrypter\" on \"project \\\"ci-gke-39e386ee-bb70\\\"\"" both failed. Final error: Error applying IAM policy for project "ci-gke-39e386ee-bb70": Error setting IAM policy for project "ci-gke-39e386ee-bb70": googleapi: Error 400: Service account service-520904206997@container-engine-robot.iam.gserviceaccount.com does not exist., badRequest[0m``` . After talking to @morgante I decided to try `active_api_identities` and the error stoped after adding it.

Reply:
ah yes you are correct we do assign `additional` roles to that identity. https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/0bcf3ca0dfebcdd99084d43eb6833bb7e55ae434/test/setup/iam.tf#L109